### PR TITLE
[BD-5][TNL-7946] Test react integration on Staff ORA views

### PR DESCRIPTION
**Pull request by giovannicimolin**
**TL;DR -** [ A short summary of what this PR does and why ]

JIRA: [TNL-7946](https://openedx.atlassian.net/browse/TNL-7946)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](./release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

**Testing Instructions**
TBD

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [OSPR-5600](https://openedx.atlassian.net/browse/OSPR-5600) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Comment by giovannicimolin** [comment]
@natabene I think I've messed up the title and the bot created an OSPR ticket instead of a blended one. :(

**Comment by natabene** [comment]
@giovannicimolin You are fine. You later added [BD-5] and bot automatically moved it to the right project and adjusted its comment in this PR. All is good.
